src/serializer/ResidualHeapSerializer.js

```diff
@@ -980,7 +980,11 @@ export class ResidualHeapSerializer {
 
   _serializeValueSymbol(val: SymbolValue): BabelNodeExpression {
     let args = [];
-    if (val.$Description) args.push(t.stringLiteral(val.$Description));
+    if (val.$Description instanceof AbstractValue) {
```

Comment:
The two more things that need to be done have nothing to do with the remaining work of serializing properties which are keyed by symbols. The two more things are strictly needed to make this pull request self-contained.